### PR TITLE
Add instructions to install and build on macOS

### PR DESCRIPTION
**Pull request by oderewonko**
- Clean the Markdown format
- Add instructions to build the .NET business DLL on macOS
- Move up the instructions to install Node.js and Yarn for the web client since it would prevent us from running Yarn if not installed yet

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
 To launch the global application, you need:
-- docker [https://www.docker.com/](https://www.docker.com/)
-- docker-compose [https://docs.docker.com/compose/install/](https://docs.docker.com/compose/install/)
+- [Docker](https://www.docker.com/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
 
-## Requirements on macOS
+### Requirements on macOS
+
+On macOS, you'll need specific steps to install Docker, Docker Compose, and the .NET Core SDK.
```
Comment (cquennouelle) [inline README.md:11]: On macOS, specific steps are needed to install Docker, Docker Compose, and the .NET Core SDK.

File: README.md [hunk 2]
```diff
@@ -17,9 +19,10 @@ For Docker, you'll need:
...
 
-1. [optional] [Homebrew](https://brew.sh)
-2. Install Docker (e.g. with Homebrew: `brew cask install docker`)
-3. Run `open /Applications/Docker.app`
+1. [optional] Install [Homebrew](https://brew.sh) to get all the required tools easily.
+2. Install Docker and Docker Compose (e.g. with Homebrew: `brew cask install docker`).
+3. Install the [.NET Core SDK](https://dotnet.microsoft.com/download#macos) (e.g. with Homebrew: `brew cask install dotnet-sdk`).
+4. Run the command `open /Applications/Docker.app`.
 
 You can then proceed with the *Install* step below.
```
Comment (cquennouelle) [inline README.md:27]: Then, proceed with the *Install* step below.

File: README.md [hunk 3]
```diff
@@ -31,49 +34,60 @@ to Install all the stack, you should execute into the root folder the line below
...
 ## Usage
 
-### Orthanc server
+### Orthanc Server
 
-Once Orthanc is running, use Mozilla Firefox at URL [http://localhost:8042/](http://localhost:8042/) to interact with Orthanc. The default username is `barbecuehealth` and its password is `barbecuehealth`.
+Once Orthanc is running, use Mozilla Firefox at URL <http://localhost:8042/> to interact with Orthanc. The default username is `barbecuehealth` and its password is `barbecuehealth`.
 
-If you want to change login/password, edit file `config/orthanc.json` and add your new login into RegisteredUsers
+If you want to change the login/password, edit the file `config/orthanc.json` and add your new login into `RegisteredUsers`.
```
Comment (cquennouelle) [inline README.md:45]: To change the login/password, edit file `config/orthanc.json` and add a new login into `RegisteredUsers`.

File: README.md [hunk 4]
```diff
@@ -31,49 +34,60 @@ to Install all the stack, you should execute into the root folder the line below
...
 
 ### Mongo Database
 
-A web Mongo client is available on the stack, you just need to use [http://localhost:8043/](http://localhost:8043/) to access MongoDB.
+A web Mongo client is available on the stack, you just need to use <http://localhost:8043/> to access MongoDB.
```
Comment (cquennouelle) [inline README.md:49]: A web Mongo client being available on the stack, just use <http://localhost:8043/> to access MongoDB.

File: README.md [hunk 5]
```diff
@@ -31,49 +34,60 @@ to Install all the stack, you should execute into the root folder the line below
...
+
+### Web Client
+
+It is developed in React TypeScript, and Yarn is used as the package manager/CLI.
+
+#### Requirement
+
+To develop, we need following tools:
```
Comment (cquennouelle) [inline README.md:57]: The following tools are need for development:

File: README.md [hunk 6]
```diff
@@ -31,49 +34,60 @@ to Install all the stack, you should execute into the root folder the line below
...
 
-### Web client
+- [Node.js](https://nodejs.org/en/) >= 12
+- [Yarn](https://classic.yarnpkg.com/lang/en/) ~1.20
 
-It is developed in React TypeScript and Yarn is using as package manager/cli
+On macOS, you can install those tools with Homebrew by running the command `brew install yarn`.
```
Comment (cquennouelle) [inline README.md:62]: On macOS, install these tools can be installed using Homebrew, by running the command `brew install yarn`.

File: README.md [hunk 7]
```diff
@@ -31,49 +34,60 @@ to Install all the stack, you should execute into the root folder the line below
...
 
 #### Docker
 
-It is available on [http://localhost:4000/](http://localhost:4000/)
+It is available on <http://localhost:4000/>.
 
 #### Locally
 
-you can run locally with the command:
+You can run locally with the command:
```
Comment (cquennouelle) [inline README.md:70]: The web client can be ran locally with the command:

File: README.md (hunk 1 above)
Comment (cquennouelle) [inline README.md:11]: On macOS, specific steps are needed to install Docker, Docker Compose and the .NET Core SDK.

File: README.md [hunk 8]
```diff
@@ -31,49 +34,60 @@ to Install all the stack, you should execute into the root folder the line below
 docker-compose -f "docker-compose.yml" up -d --build
 ```
 
-ps: If you have this kind of error “no matching manifest for windows/amd64 in the manifest list entries”, switch to Linux container in Docker Desktop.
+P.S. If you have the kind of error "no matching manifest for windows/amd64 in the manifest list entries", switch to **Linux container** in Docker Desktop.
```
Comment (cquennouelle) [inline README.md:37]: P.S. If the kind of error "no matching manifest for windows/amd64 in the manifest list entries" occurs, switch to **Linux container** in Docker Desktop.

**Review by cquennouelle — APPROVED** [review]

**Review by ptitdam2001 — APPROVED** [review]

**Pull request merged**